python-frontend/src/main/java/org/sonar/plugins/python/api/symbols/FunctionSymbol.java

```diff
@@ -69,6 +69,8 @@
     boolean isVariadic();
     boolean isKeywordOnly();
     boolean isPositionalOnly();
+    boolean isKeywordVariadic();
```

Comment:
I think it makes sense to add a short JavaDoc to say this returns true when you have `**param` (and similarly for `isPositionalVariadic`